87fc789d2db19db2a65cbf45b53afde5722ad1af Add missing stdio include in bsd-flock
diff --git a/openbsd-compat/bsd-flock.c b/openbsd-compat/bsd-flock.c
index 9b15d1ea..bf57389a 100644
--- a/openbsd-compat/bsd-flock.c
+++ b/openbsd-compat/bsd-flock.c
@@ -40,6 +40,10 @@
 #include <errno.h>
 #include <fcntl.h>
 
+#ifdef __serenity__
+#include <stdio.h>
+#endif
+
 int
 flock(int fd, int op)
 {
